src/main/java/com/quorum/gauge/common/QuorumNode.java

```diff
@@ -27,5 +27,14 @@
  */
 @Deprecated
 public enum QuorumNode {
-    Node1, Node2, Node3, Node4, Node5, Node6, Node7, Node8, Node9, Node10
+    Node1, Node1_PS1, Node1_PS2, Node1_PS3,
```

Comment:
wasn't this supposed to be deprecated?

Reply:
I've removed the changes to this file (as the extra constants were no longer used)